app/components/dashboard/card/graph_component.html.erb

```diff
@@ -1,18 +1,24 @@
 <div
   id="<%= id.to_s %>"
   data-period="<%= period.to_s %>"
-  class="pt-20px px-20px mt-8px mx-0px mb-16px bg-white br-4px bs-small d-lg-flex fd-lg-column
-         justify-lg-between h-lg-full w-lg-full mt-lg-0 pb-inside-avoid b-print-black w-print-16cm">
+  class="pt-20px px-20px mt-8px mx-0px grid-mb-16px bg-white br-4px bs-small d-lg-flex fd-lg-column
+         justify-lg-between h-lg-full w-lg-full mt-lg-0 pb-inside-avoid b-print-black w-print-16cm p-relative" style="padding-bottom: 24px;">
   <div>
     <%= title %>
     <%= summary %>
   </div>
-  <div class="p-relative mb-12px">
-    <div class="h-320px mr-16px mb-12px ml-13px">
+  <div class="p-relative">
+    <div class="h-320px" style="width: calc(100% + 12px); margin-left: -6px;">
```

Comment:
Is it possible to target this from some SCSS file somewhere? I remember you showed me the `scope` keyword in Dr. Rai; which I find very cool, and could be useful here.

As much as possible, I try to stay away from inline styling, since that invites us to stop colocating out styling.

Reply:
Will convert this to classes 💅